trenirovkf/trenirovkf/CharacterViewController.swift

```diff
@@ -0,0 +1,200 @@
+        super.init(nibName: nil, bundle: nil)
+        
+    }
+    
+    private lazy var nameLabel: UILabel = {
+        let label = UILabel()
+        label.text = dataSourse?.name
+        label.textColor = .white
+        label.font = UIFont(name: "Times New Roman", size: 35)
+        label.textAlignment = .left
+        label.numberOfLines = 0
+        return label
+    }()
+    
+    private var headingStatusLabel = UILabel()
+    private lazy var statusLabel: UILabel = {
+        let label = UILabel()
+        label.text = dataSourse?.status.rawValue
+        label.textColor = .white
+        label.font = UIFont(name: "Times New Roman", size: 21)
+        label.textAlignment = .left
+        label.numberOfLines = 0
+        return label
+    }()
+    
+    private var headingSpeciecAndGennderLabel = UILabel()
+    private lazy var speciecAndGennderLabel: UILabel = {
+        let label = UILabel()
+        label.text = "\(dataSourse?.gender?.rawValue ?? ""), \(dataSourse?.species.rawValue ?? "")"
+        label.textColor = .white
+        label.font = UIFont(name: "Times New Roman", size: 21)
+        label.textAlignment = .left
+        label.numberOfLines = 0
+        return label
+    }()
+    
+    private var headingLocationLabel = UILabel()
+    private lazy var locationLabel: UILabel = {
+        let label = UILabel()
+        label.text = dataSourse?.location.name
+        label.textColor = .white
+        label.font = UIFont(name: "Times New Roman", size: 21)
+        label.textAlignment = .left
+        label.numberOfLines = 0
+        return label
+    }()
+    
+    private var headingFirstSeenLabel = UILabel()
+    private lazy var firstSeenLabel: UILabel = {
+        let label = UILabel()
+        label.text = dataSourse?.episode?.first
+        label.textColor = .white
+        label.font = UIFont(name: "Times New Roman", size: 21)
+        label.textAlignment = .left
+        label.numberOfLines = 0
+        return label
+    }()
+    
+    private var headingEpisodesLabel = UILabel()
+    private lazy var episodesLabel: UILabel = {
+        let label = UILabel()
+        label.text = dataSourse?.episode?.first
+        label.textColor = .white
+        label.font = UIFont(name: "Times New Roman", size: 21)
+        label.textAlignment = .left
+        label.numberOfLines = 0
+        return label
+    }()
+
+    private var characterImageView = UIImageView()
+    
+    private lazy var greenIndicateView: UIView = {
+        let label = UIView()
+        label.layer.cornerRadius = 5
+        label.clipsToBounds = true
+        return label
+    }()
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+        setupConstrint()
+        view.backgroundColor = UIColor(red: 0.18, green: 0.17, blue: 0.15, alpha: 1.00)
+        
+    }
+    func setupUI() {
```

Comment:
```suggestion
   private func setupUI() {
```

так же разнеси этот метод, он слишком большой